libtsuba/src/RDG.cpp

```diff
@@ -107,9 +105,15 @@ SetTnxContextManifest(
   auto manifest_file = katana::RDGManifest::FileName(
       handle.impl_->rdg_manifest().dir(),
       handle.impl_->rdg_manifest().viewtype(), new_manifest.version());
-  handle.impl_->set_rdg_manifest(std::move(new_manifest));
 
-  txn_ctx->SetManifestInfo(new_manifest.dir(), manifest_file, new_manifest);
+  KATANA_LOG_DEBUG(
+      "new manifest version {}, dir {}, file {}", new_manifest.version(),
+      handle.impl_->rdg_manifest().dir(), manifest_file);
```

Comment:
Can you put information like this in ProgressTrace?  We don't use LOG_DEBUG much anymore.  Something like this.
```
 katana::GetTracer().GetActiveSpan().Log(
      "new manifest",
      {
          {"new_version", new_manifest.version()},
          {"manifest_dir", handle.impl_->rdg_manifest().dir()},
          {"manifest_file", manifest_file},
});
```

Reply:
Strangely I'm getting a `no matching function for call to 'katana::ProgressSpan::Log(const char [13], <brace-enclosed init ializer list>)'` with this. I can't tell any difference by eye compared to other call sites.

Reply:
More compilation error message:
```/home/yige/katana-enterprise/external/katana/libtsuba/src/RDG.cpp:117:24: error: no matching function for call to 'katana::ProgressSpan::Log(const char [13], <brace-enclosed initializer list>)'
  117 |                       });
      |                        ^
In file included from /home/yige/katana-enterprise/external/katana/libtsuba/src/RDG.cpp:36:
/home/yige/katana-enterprise/external/katana/libsupport/include/katana/ProgressTracer.h:234:16: note: candidate: 'virtual void katana::ProgressSpan::Log(const string&, const Tags&)'
  234 |   virtual void Log(const std::string& message, const Tags& tags) = 0;
      |                ^~~
/home/yige/katana-enterprise/external/katana/libsupport/include/katana/ProgressTracer.h:234:60: note:   no known conversion for argument 2 from '<brace-enclosed initializer list>' to 'const Tags&' {aka 'const std::vector<std::pair<std::__cxx11::basic_string<char>, katana::Value> >&'}
  234 |   virtual void Log(const std::string& message, const Tags& tags) = 0;
```

Reply:
Ok, I think it's the katana::Uri failed to convert itself into string in Tags's constructor. It needs explicit string().